presto-main/src/main/resources/webapp/index.html

```diff
@@ -128,6 +129,7 @@
                           queryInfo.session.source,
                           queryInfo.session.user,
                           queryInfo.state,
+                          queryInfo.errorType != null ? queryInfo.errorType : "-",
```

Comment:
Updated the index.html
But query.html still has "-" as an empty field moves all the remaining fields one level up which messes the display.